mutacc/utils/fastq_parse.py

```diff
@@ -0,0 +1,94 @@
+import gzip
+from pathlib import Path
+from contextlib import ExitStack
+
+from Bio.SeqIO.QualityIO import FastqGeneralIterator
+
+def fastq_extract(fastq_files: list, record_ids: set, dir_path = ''):
```

Comment:
I am just curious is this: dir_path = '', the common way to set a default to an empty string in python or should it be "None" or just dir_path.